src/EditorFeatures/Core/Classification/CopyPasteAndPrintingClassificationBufferTaggerProvider.Tagger.cs

```diff
@@ -48,12 +48,7 @@ private sealed class Tagger : IAccurateTagger<IClassificationTag>, IDisposable
             _subjectBuffer = subjectBuffer;
             _globalOptions = globalOptions;
 
-            // Note: because we use frozen-partial documents for semantic classification, we may end up with incomplete
```

Comment:
I was surprised that a search for "frozen" didn't hit in this document. Just wanted to make sure we are not missing a WithFrozenPartialSemantics call here.

Reply:
it happens in the classification service itself.  that said, you raise a good point that it's odd for it to happen there.  i will look into having the underlying helpers not do that, and instead bring to the feature level.